docs/team/lawrence.adoc

```diff
@@ -47,6 +47,37 @@ include::../UserGuide.adoc[tag=overduelist]
 It would be more convenient for debt collectors to view all the debtors with overdue debts with a simple
 command.
 
+=== Enhancement Added: Interest calculator
+
+==== External Behavior
+
+---
+#Start of Extract [from: User Guide]#
+
+include::../UserGuide.adoc[tag=interestCalculator]
+
+#End of Extract#
+
+---
+
+==== Justification
+
+It would be a convenient feature for debt collectors to have their debt collectors debts automatically updated.
```

Comment:
It would be convenient for debt collectors to have debtors' debts updated automatically.